ui/src/components/AppDetailToolbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Toolbar, ToolbarGroup, Button, Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { withRouter } from 'react-router-dom';
+import './AppDetailedToolbar.css';
+
+export const AppDetailToolbar = ({ onHomeClick, appName, onDisableButton, onSaveButton }) => (
+  <Toolbar className='details-toolbar'>
+    <ToolbarGroup>
+      <Breadcrumb>
+        <BreadcrumbItem onClick={onHomeClick} to='#'>Home</BreadcrumbItem>
```

Comment:
The href here should be "/" not "#" as clicking this will navigate to `http://localhost:3000/#` instead of the expected `http://localhost:3000/`. The navigation history then needs 2 back-clicks to return here.

Reply:
The reason I'm doing the route this way is that if I use `/` then the whole page re-renders whereas using `#` only rerenders the content so the navigatioin looks a lot cleaner. In fact the onClick is the logic which is doing the navigation. The reason for including `to='#'` is because it gives the button the correct styling. Without it it doesn't behave like a link.

Reply:
Okay but then it messes up the navigation history so each way has a drawack. Which is better, having the whole view re-rendered and getting the nav history to work as expected, or doing a partial re-render where the user doesn't get what they expect when they press the back button?

We could not include `to='#'` and give it a custom CSS class where we style it ourselves to behave just like a link? I'm sure if we delve into the Patternfly styling we could find it and copy the class.

Reply:
Yes probably the latter would be the better option as it would satisfy both. I'll have a look into that.